react-table/src/AdjustableTable/AdjustableTable.tsx

```diff
@@ -460,7 +460,7 @@ function Header<T>(props: React.PropsWithChildren<IHeaderProps<T>>) {
                          {...element.props}
                          setWidth={(w,min) => props.SetWidth(element.props.Key,Math.floor(w),'adjustable', min)}
                          onSort={(key, e, fld) => props.OnSort({colKey: key, colField: fld, ascending: props.Ascending},e)}
-                         sorted={props.SortKey === element.props.Key && (element.props.AllowSort ?? true)}
+                         sorted={props.SortKey === element.props.Field && (element.props.AllowSort ?? true)}
```

Comment:
This will cause issues. Columns are identified by `Key` not `Field`.

Reply:
Sortkey is set to a columns field not key though. Should we change sortkey to be set to a column's key then?

Reply:
I am confused. `SortKey` is passed in from the `prop` so that should be set in the application. Looking at SystemCenter @pdinsdale may have swapped those....

Reply:
Reverted here, you are correct, it should be up to the user to match key and sortkey and make sure that if key is different that field then those difference line up